datafusion/proto/proto/datafusion.proto

```diff
@@ -641,7 +641,7 @@ enum ScalarFunction {
   ArrayPrepend = 94;
   ArrayRemove = 95;
   ArrayReplace = 96;
-  ArrayToString = 97;
+  // 97 was ArrayToString
```

Comment:
why it was removed?

Reply:
This should be removed because this is proto for ScalarFunction.
But it seems we need to implement another version for UDF?

Reply:
added in https://github.com/apache/arrow-datafusion/pull/9113/commits/97a789abb2619fb5d2afb0f2203448f437183535